### PR TITLE
[WIP] Add diagnostic for upwards mass flux

### LINKED ISSUE #16: [FEATURE REQUEST] Vertical advective flux diagnostic
GEOS-Chem Classic has diagnostic collection AdvFluxVert to save vertical fluxes in tpcore. There is no equivalent diagnostic in GCHP. Since tpcore is in FV3 we should be able to add in an equivalent diagnostic by creating a new MAPL export in the DYNAMICS grid comp in GCHP.

### PR DESCRIPTION
**Pull request by LiamBindle**
This PR adds a diagnostic for upward mass flux of air. The standard CF name for this variable is `upward_mass_flux_of_air` (see definition here: https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html).

Marked this as a WIP because I would like to test it with GCHP 13.1 before merging (I've only tested with an old mass flux simulation that I have).

### CODE REVIEW

**Comment by LiamBindle** [comment]
This PR's WIP status is:
1. `ALLOC=.TRUE.` should be removed and  `upward_mass_flux_of_air` should only be calculated if the export is associated (re: suggestions in https://github.com/geoschem/GCHP/issues/95)
2. Waiting on updates for transport with total air rather than dry air. As it is right now `upward_mass_flux_of_air` would actually return the upwards dry air mass flux.

File: src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90 [hunk 1]
```diff
@@ -708,6 +720,23 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
...
 
+      ! Vertical motion diagnostics
+      call MAPL_GetPointer ( EXPORT, UpwardsMassFlux, 'UpwardsMassFlux', ALLOC=.TRUE., RC=STATUS )
+      _VERIFY(STATUS)
+
+      ! Get vertical mass flux
+      call fv_getVerticalMassFlux(MFXr8, MFYr8, UpwardsMassFlux, dt)
+
+      ! Flip vertical so that GCHP diagnostic is positive="up"
+      UpwardsMassFlux(:,:,:) = UpwardsMassFlux(:,:,LM:0:-1)
```
Comment (lizziel) [inline src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90:731]: In newer MAPL I believe we will be able to specify in HISTORY.rc whether to flip to positive up.  When that comes in all MAPL exports would be stored as positive down so this flipping would need to be removed.

**Comment by lizziel** [comment]
Is this PR still relevant now that vertical mass flux diagnostic is in 13.4.0?

**Comment by LiamBindle** [comment]
Closing. This is already in 13.4.0.

**Pull request closed without merging**